### DISCUSSION TITLE
Playback queue widget - what is it?

### DISCUSSION THREAD

**Discussion by Pathologic** [fooyin/fooyin#189] [General]
I don't understand the idea of playback queue widget. It looks similar to the playlist, but the behaviour is different - I can add/send tracks to the queue, change their order, but I cannot click on any track from the queue to play. All I can is to watch the tracks playing and disappearing from the queue. Maybe is's incompleted and will get some features in future, but today I don't see any benefits against playlist.

**Comment by ludouzi** [comment]
The playback queue is different from playlists in that it represents a temporary queue of tracks to be played. It takes priority over  playlists, which means the active playlist will be paused while the queue has tracks, and will be resumed once the queue is clear. 

It was added as it's how I use foobar2000 and now fooyin for library viewing/playback. I personally don't bother with playlists at all - I just use a library viewer playlist to view the current track selection and then queue up individual tracks I want to play.
